packages/esbuild/test/index.test.js

```diff
@@ -63,64 +66,103 @@ it('uses esbuild to make bundle', async () => {
 
 it('supports ignore', async () => {
   let config = {
-    checks: [{ files: fixture('big.js'), ignore: ['redux'] }]
+    checks: [{ files: fixture('cjs/big.js'), ignore: ['redux'] }]
   }
   await run(config)
-  expect(config.checks[0].size).toBe(231)
+  expect(config.checks[0].size).toBe(237)
 })
 
-it('supports custom esbuild config', async () => {
-  let config = {
-    checks: [{ config: fixture('esbuild.config.js') }],
-    configPath: ROOT_CONFIG
-  }
-  await run(config)
-  expect(config.checks[0].size).toBe(163)
-})
+describe('supports custom esbuild config', () => {
+  it('should work with commonjs config', async () => {
+    let config = {
+      checks: [{ config: fixture('cjs/esbuild.config.js') }],
+      configPath: ROOT_CONFIG
+    }
+    await run(config)
+    expect(config.checks[0].size).toBe(490)
+  })
 
-it('supports custom entry', async () => {
-  let config = {
-    checks: [{ config: fixture('esbuild.config.js'), entry: ['small'] }],
-    configPath: ROOT_CONFIG
-  }
-  await run(config)
-  expect(config.checks[0].size).toBe(66)
+  it('should work with esm config', async () => {
+    let config = {
+      checks: [{ config: fixture('esm/esbuild.config.js') }],
+      configPath: ROOT_CONFIG
+    }
+    await run(config)
+    expect(config.checks[0].size).toBe(166)
+  })
 })
 
-it('throws error on unknown entry', async () => {
-  let config = {
-    checks: [{ config: fixture('esbuild.config.js'), entry: ['unknown'] }],
-    configPath: ROOT_CONFIG
-  }
-  let err
-  try {
+describe('supports custom entry', () => {
+  it('should work with commonjs config', async () => {
+    let config = {
+      checks: [{ config: fixture('cjs/esbuild.config.js'), entry: ['small'] }],
+      configPath: ROOT_CONFIG
+    }
     await run(config)
-  } catch (e) {
-    err = e
-  }
-  expect(err).toEqual(new SizeLimitError('unknownEntry', 'unknown'))
-  expect(existsSync(config.checks[0].webpackOutput)).toBe(false)
+    expect(config.checks[0].size).toBe(229)
+  })
+
+  it('should work with esm config', async () => {
+    let config = {
+      checks: [{ config: fixture('esm/esbuild.config.js'), entry: ['small'] }],
+      configPath: ROOT_CONFIG
+    }
+    await run(config)
+    expect(config.checks[0].size).toBe(68)
+  })
+})
+
+describe('throws error on unknown entry', () => {
+  it('should work with commonjs config', async () => {
```

Comment:
```suggestion
  it('works with commonjs config', async () => {
```

and all other tests

Reply:
Fixed by [c17c150](https://github.com/ai/size-limit/pull/338/commits/c17c1507498d08ff3aa9f05cd8f20b02843f99ff)